project_test.ipynb

```diff
@@ -205,27 +193,65 @@
+      "text/plain": [
+       "<Figure size 864x360 with 3 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Plotting histograms for combined weights\n",
+    "fig, ax = plt.subplots(1, 3, figsize=(12, 5))\n",
     "\n",
-    "# Plotting Side-by-Side Histogram\n",
+    "fig.suptitle(\"Histograms showing combined weights to Student Answers in Attitude Survey\")\n",
```

Comment:
could you rename the super title to Histograms showing the Student Attitude Metric before and after performing the lab?

Reply:
Sure I updated it